### PR TITLE
Show tenure info on properties#show.

### PR DESCRIPTION
**Pull request by pudiva**

### CODE REVIEW

File: app/assets/stylesheets/hackney_property.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.hackney-property-tenure {
+  background: $govuk-focus-colour;
+  padding: 4px 8px;
+  color: white !important;
+  width: 50%;
+}
```
Comment (spannerinworks) [inline app/assets/stylesheets/hackney_property.scss:5]: from trello:
> - Colour code the displayed field green for Tenant, orange for everything else (only implement this if it's not free text)

Comment (pudiva) [reply app/assets/stylesheets/hackney_property.scss:5]: yess, but this field is kinda free text. Btw I just asked vlad to add the tenure code to the API so we can do that.

Comment (pudiva) [reply app/assets/stylesheets/hackney_property.scss:5]: will put WIP tag on the PR

**Comment by pudiva** [comment]
Still needs the list of tenures on which repairs can be raised.

**Pull request merged**